include/wayfire/config/compound-option.hpp

```diff
@@ -56,6 +57,12 @@ class compound_option_entry_base_t
         return name;
     }
 
+    /** @return The untyped default value of the tuple entry. */
+    stdx::optional<std::string> get_default_value() const
```

Comment:
> Nowadays we can use std::optional directly (not the stdx 'hack'), since the project has moved to c++17 already (for new code)

Shouldn't we make a (probably breaking) PR replacing all usages of `stdx::optional` in the project to `std::optional`?

Reply:
If you are up for that, certainly :)